### DISCUSSION TITLE
The ft60x release on crates.io points here, but the repository does not exist

### DISCUSSION THREAD

**Discussion by korken89** [NaluScientific/nalusupport#45] [General]
Hi!

I was looking at crates.io for a lib to interface with ft60x devices in my FPGA project, and found that you have released a crate for this.
This peeked my curiosity to check what you had done, but it seems the repository does not exist (or is hidden).

The documentation points to https://github.com/NaluScientific/ft60x/

Is it intentional that this should be an closed repository?

Thanks for your efforts!

**Comment by marcus-luck** [comment]
Hey @korken89 ,

Thanks for pointing it out, the repo is supposed to be public since it's just a wrapper around the d3xx driver from FTDI.

Obviously, we wrote it with a specific use in mind, and not all functionality might be there.

Ha en bra dag!
Marcus

**Reply by korken89** [reply]
Thank you for opening it! 
There is also this crate which does interfacing with FT60x via `rusb`, might be of interest as well. :)

https://github.com/apertus-open-source-cinema/ft60x-rs
